aspnetcore/hosting/iis-modules.md

```diff
@@ -0,0 +1,150 @@
+**HTTP Redirection**<br>`HttpRedirectionModule` | Yes | [URL Rewriting Middleware](xref:fundamentals/url-rewriting)
+**IIS Client Certificate Mapping Authentication**<br>`IISCertificateMappingAuthenticationModule` | Yes | 
+**IP and Domain Restrictions**<br>`IpRestrictionModule` | Yes | 
+**ISAPI Filters**<br>`IsapiFilterModule` | Yes | [Middleware](xref:fundamentals/middleware)
+**ISAPI**<br>`IsapiModule` | Yes | [Middleware](xref:fundamentals/middleware)
+**Protocol Support**<br>`ProtocolSupportModule` | Yes | 
+**Request Filtering**<br>`RequestFilteringModule` | Yes | [URL Rewriting Middleware `IRule`](xref:fundamentals/url-rewriting#irule-based-rule)
+**Request Monitor**<br>`RequestMonitorModule` | Yes | 
+**URL Rewriting**<br>`RewriteModule` | Yes† | [URL Rewriting Middleware](xref:fundamentals/url-rewriting)
+**Server Side Includes**<br>`ServerSideIncludeModule` | No | 
+**Static Compression**<br>`StaticCompressionModule` | No | [Response Compression Middleware](xref:performance/response-compression)
+**Static Content**<br>`StaticFileModule` | No | [Static File Middleware](xref:fundamentals/static-files)
+**Token Caching**<br>`TokenCacheModule` | Yes | 
+**URI Caching**<br>`UriCacheModule` | Yes | 
+**URL Authorization**<br>`UrlAuthorizationModule` | Yes | [ASP.NET Core Identity](xref:security/authentication/identity)
+**Windows Authentication**<br>`WindowsAuthenticationModule` | Yes | 
+
+†The URL Rewrite Module's `isFile` and `isDirectory` do not work with ASP.NET Core applications due to the changes in [directory structure](xref:hosting/directory-structure).
+
+## Managed Modules
+Module | .NET Core Active | ASP.NET Core Option
+--- | :---: | ---
+AnonymousIdentification | No | 
+DefaultAuthentication | No | 
+FileAuthorization | No | 
+FormsAuthentication | No | [Cookie Authentication Middleware](xref:security/authentication/cookie)
+OutputCache | No | [Response Caching Middleware](xref:performance/caching/middleware)
+Profile | No | 
+RoleManager | No | 
+ScriptModule-4.0 | No | 
+Session | No | [Session Middleware](xref:fundamentals/app-state)
+UrlAuthorization | No | 
+UrlMappingsModule | No | [URL Rewriting Middleware](xref:fundamentals/url-rewriting)
+UrlRoutingModule-4.0 | No | [ASP.NET Core  Identity](xref:security/authentication/identity)
+WindowsAuthentication | No | 
+
+## IIS Manager application changes
+When using IIS modules with .NET applications, keep in mind that when you interact with the IIS Manager to configure settings that you're directly changing the *web.config* file of the app. If you later deploy a *web.config* with the app, any changes you made in the IIS Manager on the server will be overwritten by the settings in the deployed *web.config* file. After you've made changes to an app's *web.config* on the server, mirror those changes in your project's *web.config* immediately.
```

Comment:
If you make changes to the servers *web.config* file, copy the updated *web.config file to ....